packages/nimble-components/src/table/styles.ts

```diff
@@ -89,16 +86,72 @@ export const styles = css`
         visibility: visible;
     }
 
+    .header-container {
+        display: flex;
+        align-items: center;
+        position: relative;
+    }
+
     .header-scrollbar-spacer {
         width: var(--ni-private-table-header-scrollbar-spacer-width);
     }
 
     .header {
         flex: 1;
+        overflow: hidden;
     }
 
-    .checkbox-container {
-        display: flex;
+    .column-divider {
+        border-left: 2px solid ${popupBorderColor};
+        display: none;
+        height: ${controlSlimHeight};
+        cursor: col-resize;
+        position: absolute;
+    }
+
+    .column-divider[active] {
+        display: block;
+        z-index: 1;
+    }
+
+    .header-container:hover .column-divider:not([not-active]).left,
+    .header-container:hover .column-divider:not([not-active]).right {
```

Comment:
These CSS selectors were my admittedly goofy way of dealing with not marking dividers on a column other than the one you started the drag from as `active` (and thus preventing them from appearing). I've already discussed with @mollykreis that whatever solution we come up with from preventing row hover state as you drag a divider should be applicable to other column headers, so my hope is that these CSS selectors (and ultimately the `not-active` attribute) will ultimately go away.

Reply:
Removed.